core/tests/performance_framework/perf_services.py

```diff
@@ -206,12 +206,29 @@ def get_page_timings_from_cached_session(self, page_url):
             page_session_stats=None, page_session_timings=page_session_timings)
 
     def _wait_until_page_load_is_finished(self, time_duration_secs=None):
+        """Waits for the complete page to load, otherwise XHR requests
+        made post initial page load will not be recorded.
+
+        Args:
+            time_duration_secs: int. Waiting time for page load. Defaults to
+                None.
+        """
         # Waits for the complete page to load, otherwise XHR requests
         # made post initial page load will not be recorded.
         time.sleep(time_duration_secs or self.DEFAULT_WAIT_DURATION_SECS)
 
     def _setup_proxy_server(self, downstream_kbps=None, upstream_kbps=None,
                             latency=None):
+        """Sets up a browsermobproxy server.
+        Args:
+            downstream_kbps: int. The downstream speed in kbps. Defaults to
+                None.
+            upstream_kbps: int. The upstream speed in kbps. Defaults to None.
+            latency: int. The latency of the server. Defaults to None.
+        Returns:
+            server: server. The browsermobproxy server.
+            proxy: proxy. The proxy with the specified configuration.
```

Comment:
@YashJipkate actually `return server, proxy` returns a tuple containing one term as server instance and second one as proxy. So, could you modify the type info to `tuple(server, proxy)`.

Reply:
Wasn't aware of that... Thanks!
Done!